docs/src/test-api/class-test.md

```diff
@@ -943,7 +943,7 @@ test('very slow test', async ({ page }) => {
 });
 ```
 
-Changing timeout from a slow hook:
+Changing timeout from a slow `beforeEach` or `afterEach` hook. Note that this affects the test timeout that is shared with `beforeEach`/`afterEach` hooks.
```

Comment:
what does "test timeout" mean in this line? I think it's still a bit confusing.

Reply:
I think a small note pointing out this is different (and has different behavior) to `testInfo.setTimeout` could be helpful. They are named so similarly so I think that's where I was also getting lost.